recipes/opyplus/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+- conda-forge prerelease
```

Comment:
@npavlovikj Is this enough to have the package published to the prerelease channel ? I could not find anything in the documentation

Reply:
Yeah, there is not documentation about it yet, but what you have is correct.